Tests/BDoubleTests.swift

```diff
@@ -115,8 +115,30 @@ class BDoubleTests : XCTestCase {
 
 		for (original, test, precision) in testValues
 		{
-			let result = BDouble(original)!.decimalExpansion(precisionAfterDecimalPoint: precision)
-			XCTAssert(result == test)
+			let result = BDouble(original)!.decimalExpansion(precisionAfterDecimalPoint: precision, rounded: false)
+			XCTAssertEqual(result, test)
+		}
+		
+		testValues = [
```

Comment:
This should be separated into a different test in my opinion. "testDecimalExpansionWithRounding"